docs/swagger.yaml

```diff
@@ -372,6 +372,77 @@ paths:
         in: path
         required: true
         description: The MobilePay PoS BeaconId
+  '/qr/v1/exchange':
```

Comment:
I guess the mapping to the backend service will be setup another place?

Reply:
Yes via an operation policy tied to the `operationId`